docs/en/i18n/basic-usage.md

```diff
@@ -83,13 +102,29 @@ export default {
 
 ## Specifying a root locale within an application
 
-Using the [i18n injector function](#providing-locale-data-to-i18n-aware-widgets) to surface locale properties within all i18n-aware widgets.
+Using function-based widgets and `i18n` middleware exclusively within an application means that there is no requirement to add bootstrapping code to the applications `main.ts`/`main.tsx`. To set the default locale within an application, use the `i18n` middleware from `@dojo/framework/core/middleware/i18n` and set the default locale when no locale is defined.
+
+> src/App.tsx
+
+```ts
+import { create, tsx } from '@dojo/framework/core/vdom';
+
+const factory = create({ i18n });
+
+export default factory(function App({ middleware: { i18n } }) {
+	if (!i18n.get()) {
+		i18n.set({ locale: 'en-us', rtl: false });
+	}
+	return <div>{/* the application widgets */}</div>;
+});
+```
 
-> src/main.ts
+However if the application use class-based widgets or a combination of both function-based and class-based widgets, the default locale details will need to be defined in the application registry. This can be done using the utility function, `registryI18nInjector` available from `@dojo/framework/core/mixins/I18n`.
```

Comment:
```suggestion
However, if the application uses class-based widgets or a combination of both function-based and class-based widgets, the default locale details will need to be defined in the application registry. This can be done using the utility function, `registryI18nInjector` available from `@dojo/framework/core/mixins/I18n`.
```